app/permissions/decidim/kids/permissions.rb

```diff
@@ -18,6 +19,12 @@ def permissions
               allow!
             end
           end
+        when :authorizations
+          if !user.organization.minors_participation_enabled? || user.minor?
```

Comment:
In here if minors participation is disable, no one will be able to access the authorizations. In this case we should do nothing and let the default behaviour.